### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by vlas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Running The Bot
+
+`export TELEGRAM_BOT_TOKEN='....'; export DJANGO_SETTINGS_MODULE='settings.common'; export TELEGRAM_BOT_HANDLERS='/app/telegram_bot/conversation_handlers/'; export TELEGRAM_BOT_COMMAND_SUFFIX='suf1'; python telegram_bot/bot.py`
+
+## Testing
+
+On MacOs:
+* Install system dependencies
+```commandline
+brew install py3cairo pango
```
Comment (marcserrat) [inline README.md]: guess this not needed

File: .github/build.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Build Connect Telegram Bot
```
Comment (marcserrat) [inline .github/build.yml]: File must be .github/workflows/build.yml

File: .github/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name: Deploy Connect Telegram Bot
```
Comment (marcserrat) [inline .github/deploy.yml]: same as before, you miss workflows folder

**Review by marcserrat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vlas) [reply README.md]: fixed

File: .github/deploy.yml (hunk 3 above)
Comment (vlas) [reply .github/deploy.yml]: fixed

File: .github/build.yml (hunk 2 above)
Comment (vlas) [reply .github/build.yml]: fixed

**Comment by marcserrat** [comment]
Please check https://sonarcloud.io/dashboard?id=connect-telegram-bot&pullRequest=1

will prepare pypi

**Review by marcserrat — APPROVED** [review]

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,77 @@
+[tool.poetry]
+name = "connect-reports-core"
```
Comment (jpdolz) [inline pyproject.toml]: Be careful with the name...I think is not the right one

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+version = "0.1.0"
+description = "Connect Reports Core"
```
Comment (jpdolz) [inline pyproject.toml]: Be careful, wrong description

**Review by jpdolz — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 5 above)
Comment (vlas) [reply pyproject.toml]: fixed

File: pyproject.toml (hunk 4 above)
Comment (vlas) [reply pyproject.toml]: fixed

**Review by jpdolz — APPROVED** [review]

**Pull request merged**